requirements.txt

```
@@ -0,0 +1,2 @@
+conan @ git+https://github.com/conan-io/conan.git@develop2
```

```suggestion
conan>=2.18
```
Now that we released 2.18 we can update this